Loungeware/Loungeware.yyp

```diff
@@ -856,7 +881,6 @@
   "IncludedFiles": [
     {"CopyToMask":-1,"filePath":"datafiles/games","resourceVersion":"1.0","name":"example.json","resourceType":"GMIncludedFile",},
     {"CopyToMask":-1,"filePath":"datafiles","resourceVersion":"1.0","name":"config.example.json","resourceType":"GMIncludedFile",},
-    {"CopyToMask":-1,"filePath":"datafiles","resourceVersion":"1.0","name":"config.dev.json","resourceType":"GMIncludedFile",},
```

Comment:
Will this interfere with anyone else's fork?

Reply:
Gonna have to remove those included files again